src/Symfony/Component/HttpKernel/EventListener/CacheAttributeListener.php

```diff
@@ -194,4 +200,15 @@ private function toSeconds(int|string $time): int
 
         return $time;
     }
+
+    private function hasRelevantVaryHeader(Response $response, Request $request): bool
+    {
+        // Check if the response has a Vary header that should be considered,
+        // ignoring cases where it's only 'Accept-Language' and the request has the attribute '_vary_by_language' in true.
+        return $response->hasVary()
+            && !(
+                ['Accept-Language'] === $response->getVary()
+                && $request->attributes->get('_vary_by_language')
+            );
```

Comment:
let's inline the method and the comment above?
```suggestion
        return ['Accept-Language'] === $response->getVary() ? !$request->attributes->get('_vary_by_language') : $response->hasVary()
```

Reply:
I meant this sorry:
```diff
--- a/src/Symfony/Component/HttpKernel/EventListener/CacheAttributeListener.php
+++ b/src/Symfony/Component/HttpKernel/EventListener/CacheAttributeListener.php
@@ -123,7 +123,10 @@ class CacheAttributeListener implements EventSubscriberInterface
 
         unset($this->lastModified[$request]);
         unset($this->etags[$request]);
-        $hasVary = $this->hasRelevantVaryHeader($response, $request);
+
+        // Check if the response has a Vary header that should be considered, ignoring cases where
+        // it's only 'Accept-Language' and the request has the '_vary_by_language' attribute
+        $hasVary = ['Accept-Language'] === $response->getVary() ? !$request->attributes->get('_vary_by_language') : $response->hasVary();
 
         foreach (array_reverse($attributes) as $cache) {
             if (null !== $cache->smaxage && !$response->headers->hasCacheControlDirective('s-maxage')) {
@@ -200,13 +203,4 @@ class CacheAttributeListener implements EventSubscriberInterface
 
         return $time;
     }
-
-    /**
-     * Check if the response has a Vary header that should be considered
-     * ignoring cases where it's only 'Accept-Language' and the request has the attribute '_vary_by_language' in true.
-     */
-    private function hasRelevantVaryHeader(Response $response, Request $request): bool
-    {
-        return ['Accept-Language'] === $response->getVary() ? !$request->attributes->get('_vary_by_language') : $response->hasVary();
-    }
```